Summers_James/app_base.htm

```diff
@@ -0,0 +1,150 @@
+                        <li class = " ${ 'active1' if request.dmp.page == 'terms' else ''} " tabindex=4><a class="paddingright" href= "/terms" >Terms</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'faq' else ''} " tabindex=5><a class="paddingright" href="/faq">FAQ</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'catalog' else ''} " tabindex=6><a class="paddingright" href="/catalog">Catalog</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'list' else ''} " tabindex=7><a class="paddingright" href="/catalog/list">Products</a></li>
+                        <li class = " ${ 'active1' if request.dmp.page == 'manager' else ''} " tabindex=9><a class="paddingright" href="/manager/manager">Manager</a></li>
+                    </ul>
+        
+                        <ul class="nav navbar-nav navbar-right">
+                            % if request.user.is_authenticated:
+                            <li class="accountDisplay">hello ${request.user}</li>
+                            <li class="dropdown">                          
+                                    <a href="#" class="nav navbar-nav navbar-right dropdown-toggle" type="button" data-toggle="dropdown" tabindex=10>Account
+                                      <span class="caret"></span></a>
+                                      <ul class="dropdown-menu">
+                                        <li class = " ${ 'active1' if request.dmp.page == 'logout' else ''} "><a href="/account/logout">Logout<i class="icon-user icon-white"></i></a></li> 
+                                        <li><a href="#">My Cart</a></li>
+                                        <li class="divider"></li>
+                                        <li><a href="#">My Account</a></li>
+                                      </ul>
+                                  </li>
+                            % else :
+                                <li class = " ${ 'active1' if request.dmp.page == 'signup' else ''} "><a href="/account/signup">Sign-Up<i class="icon-user icon-white"></i></a></li>                               
+                                <li class = " ${ 'active1' if request.dmp.page == 'login' else ''} "><a href="/account/login">Login<i class="icon-user icon-white"></i></a></li> 
+                            % endif 
+                        </ul>
+                    </div>
+                  </nav>
+        </%block>
+            
+    <%block name="catalogList">
+    <div class="myListTable" class="relative">
+        <div class="displayCategoryOuter">
+            <ul class="displayCategoryInner">
+                    <li><a href="/catalog/index/">All products</a></li>
+                    % for cat in categories:
+                    <li class = " ${ 'activeUnderline' if cat.id == underline else ''} "><a href="/catalog/index/${cat.id}">${cat.name}</a></li>
+                    % endfor
```

Comment:
Clever way to have the category of the product highlighted without having to use JS!

Reply:
:)